### DISCUSSION TITLE
ServiceStack auth with SMS/Email VerfiyCode

### DISCUSSION THREAD

**Discussion by mylvghb** [ServiceStack/Discuss#92] [Ideas]
There are many sites that now offer SMS/Email VerifyCode login without the need to log in with a username and password. can ServiceStack add that feature?

**Comment by mythz** [comment]
From [ServiceStack v8](https://docs.servicestack.net/releases/v8_00) ServiceStack just adopts ASP.NET Core Identity UI in its new templates. Any additional Auth features like this would need to have an official ASP .NET Core implementation first before they'll be added to the project templates.

This also means if anyone else as implemented this feature in their ASP.NET Core App's, you'll also be able to easily add it to new ServiceStack Apps.
